app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts
+
+  def three_recent_posts
+    posts.limit(3)
```

Comment:
- [ ] @Yosefgeda great work adding a method to return the most recent posts, however your method doesn't guarantee you will get the 3 recent post, kindly consider changing it , take a look at my suggestion below
```suggestion
    `posts.order('created_at Desc').limit(3)`
```